scss/_variables.scss

```diff
@@ -1568,6 +1568,7 @@ $modal-scale-transform:             scale(1.02) !default;
 // Define alert colors, border radius, and padding.
 
 // scss-docs-start alert-variables
+$alert-color:                       $black !default; // Boosted mod
```

Comment:
why not use inherit as a value here? and then use this variable inside alert.scss?

Reply:
Yes indeed, that's better